samples/FleetProvisioning/README.md

```diff
@@ -306,7 +319,144 @@ aws iot create-provisioning-claim \
 Finally, you can also pass the certificate signing request while invoking the Fleet Provisioning sample.
 
 ``` sh
-mvn compile exec:java -pl samples/FleetProvisioning -Dexec.mainClass="fleetprovisioning.FleetProvisioningSample" -Dexec.args="--endpoint <endpoint>
+mvn compile exec:java -pl samples/FleetProvisioning -Dexec.mainClass="fleetprovisioning.Mqtt5FleetProvisioningSample" -Dexec.args="--endpoint <endpoint>
 --cert <path to certificate> --key <path to private key> --template_name <template name> --template_parameters '{\"SerialNumber\":\"1\",\"DeviceLocation\":\"Seattle\"}'  --csr <path to csr file>"
 ```
 
+## Service Client Notes
+### Difference relative to MQTT311 IotIdentityClient (Fleet Provisioning)
```

Comment:
This README seems to represent a single file that's not specifically Mqtt3 or Mqtt5 client focused. We may want to rename this to something like "Differences between MQTT5 and MQTT311 IotIdentityClients (Fleet Provisioning)"